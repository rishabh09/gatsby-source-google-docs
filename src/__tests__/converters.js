const data = require(`./data.json`)

const {
  convertGoogleDocumentToJson,
  convertJsonToMarkdown,
} = require("../utils/converters")

test("Converts document", () => {
  const file = {id: `hi`, metadatIsCool: `yeah`}
  const content = convertGoogleDocumentToJson(data)
  expect(convertJsonToMarkdown({file, content})).toMatchSnapshot()
})
