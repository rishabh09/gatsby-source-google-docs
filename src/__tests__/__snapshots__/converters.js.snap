// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Converts document 1`] = `
"---
id: hi
metadatIsCool: yeah

---
  
## headers

# Header 1

## Header 2

### Header 3

#### Header 4

##### Header 5

## Indented text


Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aliquam hendrerit mi posuere lectus. Vestibulum enim wisi, viverra nec, fringilla in, laoreet vitae, risus.




## Code


Markdown.generate();

## Lists


 1. Red
 2. Green
 3. Blue





 1. Buy flour and salt
 2. Mix together with water
 3. Bake 

## Links


This is [_an example_](http://example.com/) link.

## Inline formatting


_Italics_ and **bold** and **_bold italics_** and some _underlined text_ and ~~striked out~~ 


This paragraph has some code in it.

## Images

![Image Title](https://lh3.googleusercontent.com/Z4q7bVeLnbi4CpZYOCMAOU4_H0uovwNvElapqBEC1DVp93TXFVpyFtCZG6xj_j3l85XsTsj7zDxpoiqWDZj0-DmIUeBAazlC2a59Z7uJqPWrwDegGxBt_pKp5Qj2yeYgVSxGz-61sBmuP7kfxw)



"
`;
